server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/Capability.scala

```diff
@@ -86,20 +90,19 @@ final case class MailCapabilityProperties(maxMailboxesPerEmail: MaxMailboxesPerE
                                           maxSizeMailboxName: MaxSizeMailboxName,
                                           maxSizeAttachmentsPerEmail: MaxSizeAttachmentsPerEmail,
                                           emailQuerySortOptions: List[EmailQuerySortOption],
-                                          mayCreateTopLevelMailbox: MayCreateTopLevelMailbox) extends CapabilityProperties {
-}
+                                          mayCreateTopLevelMailbox: MayCreateTopLevelMailbox) extends CapabilityProperties
 
-final case class QuotaCapabilityProperties() extends CapabilityProperties {
-}
+final case class QuotaCapabilityProperties() extends CapabilityProperties
 
 final case class QuotaCapability(properties: QuotaCapabilityProperties = QuotaCapabilityProperties(),
-                                 identifier: CapabilityIdentifier = JAMES_QUOTA) extends Capability {
-}
+                                 identifier: CapabilityIdentifier = JAMES_QUOTA) extends Capability
 
-final case class SharesCapabilityProperties() extends CapabilityProperties {
-}
+final case class SharesCapabilityProperties() extends CapabilityProperties
 
 final case class SharesCapability(properties: SharesCapabilityProperties = SharesCapabilityProperties(),
-                                  identifier: CapabilityIdentifier = JAMES_SHARES) extends Capability {
-}
+                                  identifier: CapabilityIdentifier = JAMES_SHARES) extends Capability
+
+final case class VacationResponseCapabilityProperties() extends CapabilityProperties
```

Comment:
case object ?

Reply:
You did similar things with @rouazana regarding `QuotaCapabilityProperties` and `SharesCapabilityProperties`. I'm not expert after but if I remember it had to do with the serialization, as we wanted to have those being sent back as part of the session response even if they are empty?

Reply:
Ok, keep it like that for the moment. I will see if i can refactor all those empty case classes later.

Reply:
Yeah sorry I tried to check quickly but couldn't really get to anything